Branch-SDK/src/main/java/io/branch/coroutines/InstallReferrers.kt

```diff
@@ -230,6 +230,98 @@ suspend fun getXiaomiGetAppsReferrerDetails(context: Context): InstallReferrerRe
     }
 }
 
+suspend fun getMetaInstallReferrerDetails(context: Context): InstallReferrerResult? = withContext(Dispatchers.Default) {
+    try {
+        val fbAppID = context.getString(
+            context.resources.getIdentifier("facebook_app_id", "string", context.packageName)
```

Comment:
I recommend we do not make any assumptions about the string resource id and instead expect a function call to set this explicitly.